src/routes/Pool/components/PoolCard.tsx

```diff
@@ -49,7 +80,7 @@ export const PoolCard = () => {
           </Typography>
           <Button
             variant="contained"
-            color="primary"
+            color="primary" disabled={data?.vipAmountUser.isLessThan(data.vipAmount)}
```

Comment:
good use of the `?` operator. Have used too much legacy JS to remember this is bundled in TS. Makes life so much easier.